web/next.config.js

```diff
@@ -17,8 +18,10 @@ const nextConfig = {
             mergeDefaultDirectives: true,
             "img-src": [
               "'self'",
+              "blob:", // Used to enforce image width and height
```

Comment:
review with security team to understand any potential concerns

Reply:
resolved with security